vault/seal_autoseal.go

```diff
@@ -18,16 +21,20 @@ import (
 // applicable in the OSS side
 var barrierTypeUpgradeCheck = func(_ string, _ *SealConfig) {}
 
+const sealHeathTestInterval = 1 * time.Minute
```

Comment:
Didn't we want to check this hourly?

Reply:
Yeah, I'm back and forth on that.  Since it's a cheap encrypt/decrypt I coded it to be more frequent.  Open to thoughts.

Reply:
I'm assuming this is being done on all auto-unseal implementations, not just pkcs11 implementations. If that is the case could the costs associated with using the various KMS providers within the cloud start adding up? I don't have a good sense of those costs across the various implementations.

Reply:
`sealHeathTestInterval` - should this be sealHea**l**thTestInterval?

Reply:
It sure should.